src/autoscaler/pruner/config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/cloudfoundry-incubator/candiedyaml"
+)
+
+const (
+	DefaultLoggingLevel        = "info"
+	DefaultIntervalInHours int = 24
+	DefaultCutoffDays      int = 30
+)
+
+type LoggingConfig struct {
+	Level string `yaml:"level"`
+}
+
+var defaultLoggingConfig = LoggingConfig{
+	Level: DefaultLoggingLevel,
+}
+
+type DbConfig struct {
+	MetricsDbUrl string `yaml:"metrics_db_url"`
+}
+
+type PrunerConfig struct {
+	IntervalInHours int `yaml:"interval_in_hours"`
+	CutoffDays      int `yaml:"cutoff_days"`
```

Comment:
I have a feeling that the cutoff will be different per DB but we can leave this here and fix it once we add a second DB/Table.